app/controllers/api/bookings_controller.rb

```diff
@@ -1,6 +1,11 @@
 class Api::BookingsController < ApplicationController
   protect_from_forgery with: :null_session
 
+  def index
```

Comment:
There is API is not authenticated. Could you please add logic so that only authenticate user would be able to call this API?